packages/amplify-cli/src/lib/attach-backend-steps/a10-queryProvider.js

```diff
@@ -17,6 +17,12 @@ async function getProvider(context, providerPlugins) {
   let result;
   const providers = [];
   const providerPluginList = Object.keys(providerPlugins);
+
+  if (providerPluginList.length === 0) {
+    const errorMessage = 'Found no provider plugins';
+    throw new Error(errorMessage);
```

Comment:
We should give customers some next steps to mitigate this issue like - "Run `amplify plugin scan` for scanning provider plugins in your system".